### DISCUSSION TITLE
rdb kernel module for (at least) firmware-nanopi6  and firmware-nanopi5

### DISCUSSION THREAD

**Discussion by tarastella** [MichaIng/DietPi#6402] [Q&A] [answered]
I would like to run ceph on both nanopi R6S and R5S. 
The rdb kernel module is missing from firmware-nanopi6 and firmware-nanopi5
Is it possible to include it in the next releases?
Thanks, Tara

Edit: 
Need to enable CONFIG_BLK_DEV_RBD=m
I did it on a vanilla kernel 5.10.110 using /proc/config.gz (generated with modprobe config) and compiled only the rdb.ko module

**Comment by MichaIng** [comment, marked as answer]
Answered here: https://github.com/MichaIng/DietPi/discussions/6403#discussioncomment-6065369

And to check for that option in particular: https://github.com/friendlyarm/kernel-rockchip/blob/nanopi5-v5.10.y_opt/arch/arm64/configs/nanopi6_linux_defconfig
Indeed it is missing.
